.htaccess

```diff
@@ -56,12 +61,6 @@ DirectoryIndex index.php index.html index.htm
 <IfModule mod_rewrite.c>
   RewriteEngine on
 
-  # Force HTTPS for site vistors.
-  # Added 2016-10-19 ME
-  RewriteCond %{HTTPS} off [OR]
-  RewriteCond %{HTTP_HOST} ^www.librariesoforegon\.org*
-  RewriteRule ^(.*)$ https://librariesoforegon.org/$1 [L,R=301]
-
```

Comment:
I think we should keep this https customizations so that they continue to work in production